Sources/Basics/HTTPClient.swift

```diff
@@ -80,6 +82,7 @@ public struct HTTPClient {
 
     private func _execute(request: Request, requestNumber: Int, callback: @escaping (Result<Response, Error>) -> Void) {
         if self.shouldCircuitBreak(request: request) {
+            diagnosticsEngine.emit(warning: "Circuit breaker triggered for \(request.url)")
```

Comment:
Right now this is the right way to do it.  I have unfinished changes to DiagnosticsEngine to allow all references such as URLs and File paths to be passed as separate parameters, so that they can show up nicely in various clients (either with formatting, or for file paths, as clickable links etc).  Just a FYI but for now this is good.

Reply:
that sounds like a very nice enhancemnt